src/controllers/accounts/AccountsAssetsController.ts

```diff
@@ -0,0 +1,65 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+import { BadRequest } from 'http-errors';
+
+import { validateAddress } from '../../middleware';
+import { AccountsAssetsService } from '../../services/accounts';
+import AbstractController from '../AbstractController';
+
+export default class AccountsAssetsController extends AbstractController<AccountsAssetsService> {
+	constructor(api: ApiPromise) {
+		super(api, '/accounts/:address/', new AccountsAssetsService(api));
+	}
+
+	protected initRoutes(): void {
+		this.router.use(this.path, validateAddress);
+
+		this.safeMountAsyncGetHandlers([
+			['asset-balances', this.getAssetBalances],
+			['asset-approvals', this.getAssetApprovals],
+		]);
+	}
+
+	private getAssetBalances: RequestHandler = async (
+		{ params: { address }, query: { at, assets } },
+		res
+	): Promise<void> => {
+		const hash = await this.getHashFromAt(at);
+
+		let assetsArray: number[] = [];
+		if (typeof assets === 'string') {
+			assetsArray = this.parseQueryParamArrayOrThrow(
+				assets,
+				'assets query parameter is not in the correct format.'
```

Comment:
```suggestion
				'`assets` query parameter is not in the correct format. Should be of the form `?assets[]=42&assets[]=3`'
```
Maybe? It at least gives a clue to the person reading the code, but maybe we should just say to reference the docs.

Reply:
I like this as well, I think what you have suggested is good, then we can tag on at the and 'reference the docs for more information.'